mw-webapp/src/dataAccessLogic/renderCellValue/renderCellItem.tsx

```diff
@@ -0,0 +1,110 @@
+import {useState} from "react";
+import {Input} from "src/component/input/Input";
+import {updateCell} from "src/dataAccessLogic/renderCellValue/helpers/updateCell";
+import {renderCellSpan} from "src/dataAccessLogic/renderCellValue/renderCellSpan";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+
+/**
+ * Properties of {@link DayReport} that have type string[] for dynamic update
+ */
+export interface ColumnNameProps {
+
+  /**
+   * Student comments
+   */
+  studentComments: string[];
+
+  /**
+   * Learned for today
+   */
+  learnedForToday: string[];
+}
+
+/**
+ * Cell item props
+ */
+interface CellItemProps {
+
+  /**
+   * Cell item's text
+   */
+  content: string;
+
+  /**
+   * Element of custom arrays
+   */
+  arrayItem?: JobDone | PlanForNextPeriod | CurrentProblem | MentorComment;
+
+  /**
+   * Parent uuid for cells with type string
+   */
+  parentUuid?: string;
+
+  /**
+   * Column name for cells with type string[]
+   */
+  columnName?: keyof ColumnNameProps;
+
+  /**
+   * Index of element in string array
+   */
+  index?: number;
+
+  /**
+   * Value isDone for different styles
```

Comment:
why does cells contains such prop `isDone`? Looks like mistake

Reply:
Don't know how to change it right now. Because styles of content in cell depends on this value. 
If I rename this property to isContentDone? will be better?